Preference Access SSN Matching/REPWRITERSPECS/RB.PREFACCESS.SSNMATCHING

```diff
@@ -0,0 +1,1628 @@
+**    a list of up to 300 alternate accounts for any given home account.
+**    a. For each alternate account, look for the SSN. If the SSN is not
+**       found at the account level, then continue the search with any
+**       eligible Shares, Loans and External Loans.
+** 5. For each match found, look through the PA list
+**    a. If there is already an existing PA record pointing to the
+**       target Account, Share, Loan or External Loan, mark the PA
+**       record status as VALID.
+**    b. If a PA match is not found, add a new PA record to the list and
+**       mark its� status as NEW.
+** 6. Loop through the final list of PA records.
+**    a. If the record�s current status is UNKNOWN process the record
+**       further based upon the CFG Letter file parameter.
+**      i. If the invalid Preference Access Records Processing parameter
+**         is set to DELETE, change the record�s status to INVALID
+**      ii. If the parameter is set to IGNORE then set the record�s
+**          status to IGNORE
+**      iii. If the parameter is set to VALIDATE then validate that the
+**           PA record points to a valid account and, as appropriate a
+**           valid Share, Loan or External Loan and if so, mark the
+**           record�s status as VALID otherwise, change the record�s
+**           status to INVALID.
+**    b. If the record�s status is VALID and the access flags (WD,
+**       Inquiry, Deposit) do not match the related settings in the CFG
+**       Letter file, mark the record for UPDATE.
+**    c. Process the PA record based upon its� final record status.
+**      i. If the status is NEW, create a new PA record.
+**      ii. If the status is UPDATE, update the existing PA record so
+**          the access flags match the current CFG Letter file settings.
+**      iii. If the status is INVALID, delete the PA record.
+**      iv. If the status is IGNORE, leave the PA record as-is.
+**    d. Update the report with the account and PA list information
+**       along with the action taken for each of the PA records.
+**    e. Output the necessary data to the FM report to be used by Misc.
+**       Processing to update the member�s accounts.
+**
+** <PROMPTS>
+**   There are no batch prompts as the program's run-time parameters are
+**   set in a configuration Letter file.
+**
+** <INPUT>
+**   <LETTERFILE>
+**     PREFACCESS.SSNMATCHING.CFG
+**       Program configuration parameters. Instructions for setting
+**       parameters are within the Letter file.
+** <OUTPUT>
+**   <REPORTS>
+**     Pref Access SSN Update RPT
+**       Summary Report
+**     Pref Access SSN Update FM
+**       Used by Misc. Processing to update the member's accounts
+**
+** <SUPPORT FILES>
+**
+** <FM PERFORMED>
+**   Preference Access records created, updated and/or deleted
+**
+** <ADDITIONAL INFO>
+**   The program can process up to 200 alternate accounts for each
+**   home account found and up to
```

Comment:
missing number after "up to"

Do you mean it can process up to 300 alternate accounts for home account and up to 200 alternate accounts for each Target Account?

Reply:
For any given "home" account there can be up to (& including) 200 different alternate accounts. There can be up to (&incl)  300 PA records for each home account (not 300 PA records per alternate account.)

Clarified/corrected in the program header.